xarray/tests/test_dataset.py

```diff
@@ -2061,7 +2061,7 @@ def test_rename(self):
             assert_equal(Variable(dims, v.values, v.attrs),
                          renamed[k].variable.to_base_variable())
             assert v.encoding == renamed[k].encoding
-            assert type(v) == type(renamed.variables[k])  # noqa: E721
+            assert isinstance(v, type(renamed.variables[k]))  # noqa: E721
```

Comment:
If you ran `autopep8` on this - it did a bit too much! The lines with `noqa` should be ignored (and existing code won't function correctly with the changes)

The changes above re line breaks around operators are fine either way

Reply:
I did run autopep8. I will fix this tomorrow

Reply:
Great, only two items need manual reverting I think (i.e. those with noqa). Not sure why autopep8 didn't spot those